### ISSUE TITLE
Pretrained model available?

### ISSUE THREAD

**Issue by sebo313** [switchablenorms/DeepFashion2#40]
Great work! Is there any pretrained model available to run inference on my own data? Or how can i run the [pycocoDemo.ipynb](https://github.com/switchablenorms/DeepFashion2/blob/master/deepfashion2_api/PythonAPI/pycocoDemo.ipynb)?

**Comment by Irtza** [comment]
BUMP!

**Comment by baixueyaoyao** [comment]
BUMP!

**Comment by f-arthr** [comment]
Still not?

**Comment by bluesky314** [comment]
When is it being shared?

**Comment by Manishsinghrajput98** [comment]
> Great work! Is there any pretrained model available to run inference on my own data?

yes

**Comment by sebo313** [comment]
@Manishsinghrajput98 awesome! Are you willing to share it with the community?

**Comment by robi772** [comment]
BUMP!

**Comment by Manishsinghrajput98** [comment]
Thanks
Sorry for the inconvenience

Give me a time. In this time there is corona virus effected in India. And I
am in my hometown. And in my hometown I don't have any resources to share
with you.


Thanks and regards
Manish Rajput
Junior Machine Learning Engineer
Indore MP India

**Comment by vince-lynch** [comment]
Hey also interested in pretrained model.

**Comment by aashay96** [comment]
Will be releasing a model from our side (www.kubric.io) very soon!

**Comment by Manishsinghrajput98** [comment]
I will also upload with good accuracy very soon.

**Comment by greeneggsandyaml** [comment]
@Manishsinghrajput98 and @aashay96 thank you so much! I am looking forward to a pretrained model!

**Comment by zxpzhong** [comment]
@Manishsinghrajput98 @aashay96   I'm seeking a fasion segmentation model pretrained on Deep Fashion2. So is there a timeline to release your pretrained model? Thank you so much!

**Comment by sayalraza** [comment]
Bump

**Comment by Manishsinghrajput98** [comment]
Hello all i have released pretrained  model please follow this tutorial. this is my first blogs.   

https://manishsinghrajput96.blogspot.com/

thanks

**Comment by minushuang** [comment]
@Manishsinghrajput98  than u very much for your sharing

**Comment by Manishsinghrajput98** [comment]
Your welcome sir

**Comment by vishakhhegde** [comment]
Does anyone know why the train data only has 191K images? While the paper suggests that it should have 391K training images.
